packages/state-transition/package.json

```diff
@@ -60,7 +60,7 @@
   "dependencies": {
     "@chainsafe/as-sha256": "^0.4.1",
     "@chainsafe/bls": "7.1.3",
-    "@chainsafe/blst": "^0.2.9",
+    "@chainsafe/blst": "^0.2.10",
```

Comment:
why do we have that as a dependency, is it a peer dependency for another package?

Reply:
Of bls, [yes](https://github.com/ChainSafe/bls/blob/master/package.json#L122)